### ISSUE TITLE
Worker crashes due to JSON.parse on version 7.1.1 

### ISSUE THREAD

**Issue by julianfbeck** [LearningLocker/learninglocker#1608]
#### What version were you using?
LearningLocker: v7.1.1
Redis 4.x

#### What steps can we follow to reproduce the behaviour?
Add statements via xAPI

#### What is the actual behaviour?
The worker tries to parse a number and crashes 

#### What is the expected behavior?
No worker crash

#### Is there any additional information that will help us replicate/understand the problem?
The problem seems  to be caused by the already merged PR #1591 
When removing the change, it works perfectly

**Comment by davidstoker** [comment]
I had the same thing happen after upgrading from v6.4.0 => v7.1.1. 

I observed that there was a single left-over value under the Redis key `LEARNINGLOCKER:statement.notify` that was just a UUID value from a statement ID. I see that #1591 describes the new payload format being `{"statementId":"0e4f1541-321c-4b60-b294-dde1cc6f9f87","organisationId":"603fd8e4fce8e30141294682"}`

In my case, I had to manually delete that key in order to get the new worker to process correctly. Not sure if there was any better solution since older worker versions expected one payload format and the new version expects a different one. Any unprocessed statements from old would fail on new naturally.

I run a Docker setup with automated deployment so my assumption is that a statement came in on the older version in the midst of the new version deploying causing the mismatch.
